Web/Pages/Transactions/Transactions.razor

```diff
@@ -58,7 +59,7 @@
             @context.CounterPartyName
         </MudTd>
         <MudTd DataLabel="Amount">
-            @GetAmount(context).ToString("C")
+            @GetAmount(context).ToString("C", CultureInfo.CreateSpecificCulture("be-NL"))
```

Comment:
I'm not a fan of hardcoding this to Belgian culture, we should be able to use CurrentUICulture to detect the user's culture

Reply:
How is the CurrentUICulture taken from the user?

Reply:
See https://learn.microsoft.com/en-us/aspnet/core/blazor/globalization-localization?view=aspnetcore-7.0#dynamically-set-the-culture-from-the-accept-language-header
The browser sends the user's culture info (possibly multiple values with scores) in the `Accept-Language` header,